front-end/packages/robustar/src/views/TrainPad.vue

```diff
@@ -205,7 +205,7 @@ export default {
       console.log(res);
       this.$root.finishProcessing();
       this.$root.alert('success', 'Training started successfully');
-      window.open('http://localhost:6006');
+      window.open('http://localhost:'+(await (await fetch("serverUrls.json")).json()).tensorboardPort);
```

Comment:
This gives an error during execution because `await` can only be used within a `async` function.
Suggestion: create an async function locally and put `window.open` inside `then` block. e.g.

```
const getPort = async () => { ... }
getPort.then(port => { window.open(...) } )
```